addons/website_sale/models/product_image.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api
+
+
+class ProductImage(models.Model):
+    _name = 'product.image'
+    _description = "Product Image"
+    _inherit = ['image.mixin']
+    _order = 'product_tmpl_id, sequence, id'
+
+    name = fields.Char("Name", required=True)
+    sequence = fields.Integer(default=10, index=True)
+
+    image_original = fields.Binary(required=True)
```

Comment:
why not keep image_original from image.mixin?

Reply:
It's just to make it required on that model, as it doesn't make sense to add product.images in the o2m if those don't have any image on them.

But it can't be required on the mixin of course since it's used on template and variant.